dnf/artifact-registry.py

```diff
@@ -0,0 +1,60 @@
+#   manpage for plugin(s)
+#   config file
+
+import dnf
+
+import google.auth
+import google.auth.transport.requests
+from google.auth import compute_engine
+from google.auth.exceptions import DefaultCredentialsError, RefreshError
+from google.oauth2 import service_account
+
+
+class ArtifactRegistry(dnf.Plugin):
+  """DNF Plugin for authenticated access to Google Artifact Registry"""
+
+  name = "artifact-registry"
+
+  def __init__(self, base, cli):
+    super(ArtifactRegistry, self).__init__(base, cli)
+    self.base = base
+    self.credentials = self._get_creds()
+
+  def _get_creds(self):
+    cp = self.read_config(self.base.conf)
```

Comment:
Sorry if this is obvious, but what does `cp` stand for in this context?

Reply:
configparser

Reply:
Can we rename it to parser? I think it would read easier.